main/f2fs-tools/APKBUILD

```diff
@@ -1,29 +1,23 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=f2fs-tools
-pkgver=1.12.0
+pkgver=1.13.0
 pkgrel=0
 pkgdesc="Tools for the Flash-Friendly File System (F2FS)"
 url="http://git.kernel.org/cgit/linux/kernel/git/jaegeuk/f2fs-tools.git"
 arch="all"
 license="GPL-2.0-or-later LGPL-2.1-or-later"
-depends=""
-depends_dev=""
 makedepends="$depends_dev automake autoconf bsd-compat-headers
 	libtool util-linux-dev linux-headers"
-install=""
 subpackages="$pkgname-doc $pkgname-dev $pkgname-libs"
-source="https://git.kernel.org/cgit/linux/kernel/git/jaegeuk/f2fs-tools.git/snapshot/$pkgname-$pkgver.tar.gz
+source="https://git.kernel.org/cgit/linux/kernel/git/jaegeuk/f2fs-tools.git/snapshot/f2fs-tools-$pkgver.tar.gz
 	"
-builddir="$srcdir/$pkgname-$pkgver"
 
 prepare() {
 	default_prepare
-	cd "$builddir"
```

Comment:
packages in main/ don't lose this.

https://gitlab.alpinelinux.org/Leo/atools/blob/master/apkbuild-lint#L298

Reply:
Not sure I understand the rational behind this but I've added them back in.

Reply:
main/ is 2 year support which has versions of  Alpine that  do not support it.

Reply:
Thanks for the explanation